### ISSUE TITLE
Duplicates when using multiple indexes.

### ISSUE THREAD

**Issue by jnssnmrcs** [skoshx/pentagon#24]
When there are more than one index key for a table the current code in whereToKeys simply uses the returned items from kv.getMany, which will return duplicate items, one for each index key. However I think it only occurs when passing multiple properties that has an index of any type in the where-args.

demonstrated by code below:

```ts
const kv = await Deno.openKv();

const user = {
  id: "1",
  favoriteColor: "blue",
  email: "my.email@proton.me",
};

const primaryKey = ["users", "1"];
const uniqueKey = ["users_by_email", "my.email@proton.me"];
const nonUniqueKey = ["users_by_color", "blue", "1"];

const res = await kv
  .atomic()
  .check({ key: primaryKey, versionstamp: null })
  .check({ key: uniqueKey, versionstamp: null })
  .set(primaryKey, user)
  .set(uniqueKey, user)
  .set(nonUniqueKey, user)
  .commit();

if (res.ok) {
  const items = await kv.getMany([primaryKey, uniqueKey, nonUniqueKey]);

  console.log(items);
  // PRINTS:
  // [
  //   {
  //     key: [ "users", "1" ],
  //     value: {
  //       id: "1",
  //       favoriteColor: "blue",
  //       email: "my.email@proton.me"
  //     },
  //     versionstamp: "00000000000000010000"
  //   },
  //   {
  //     key: [ "users_by_email", "my.email@proton.me" ],
  //     value: {
  //       id: "1",
  //       favoriteColor: "blue",
  //       email: "my.email@proton.me"
  //     },
  //     versionstamp: "00000000000000010000"
  //   },
  //   {
  //     key: [ "users_by_color", "blue", "1" ],
  //     value: {
  //       id: "1",
  //       favoriteColor: "blue",
  //       email: "my.email@proton.me"
  //     },
  //     versionstamp: "00000000000000010000"
  //   }
  // ]
}

```

**Comment by skoshx** [comment]
Hello, is this closed by the latest Pull Requests?

**Comment by jnssnmrcs** [comment]
No. There's a test for it but it's ignored since it fails.

**Comment by skoshx** [comment]
Oh shoot, okay! I will for now make a commit that adds it back, so the CI
will show the reality, and then create a release when this, and the
updating bug is fixed :)

**Comment by skoshx** [comment]
Closed by [e682eb8](https://github.com/skoshx/pentagon/commit/e682eb8437eefe586d496d9174caf537099a0662)
